x/concentrated-liquidity/types/constants.go

```diff
@@ -11,6 +11,8 @@ import (
 const (
 	// Precomputed values for min and max tick
 	MinInitializedTick, MaxTick int64 = -108000000, 342000000
+	MinInitializedTickV2        int64 = -270000000
+	MinCurrentTickV2            int64 = MinInitializedTickV2 - 1
```

Comment:
Is this V2 distinction mainly to keep tests compatible?

Reply:
Yes, "V2" notation is temporary. By the end of this refactor, anything "V2" will become the canonical.

We will still need to keep track of V1 threshold for deciding on what tick-to-price and price-to-tick conversation strategy to use. Note that for all pools, anything above the old threshold will use 18 decimal strategies and anything below the threshold 36 decimal strategies.

It will probably make sense to rename V1 `MinInitializedTick` to `TickConversionThreshold`. Similarly with V1 `MinSpotPrice`